### ISSUE TITLE
Automated Confirmations, Notifications and Alerts

### ISSUE THREAD

**Issue by brianblain** [brianblain/TBS#9]
"Leave your email address here if you would like to be notified when your recurring event ends"

**Comment by rdbrinkhurst** [comment]
The only way to do this would be to force the user to sign up before putting inputting their data.

**Comment by brianblain** [comment]
yes - we discussed that early on.

**Comment by rdbrinkhurst** [comment]
Right, I remember that you didn't want user to have to sign up in order to put in a calendar entry. Are you now asking for users to have to sign up? Again, we can discuss this over the phone if it's easier.
